### ISSUE TITLE
GUI crash when attempting to sorting by MD column

### ISSUE THREAD

**Issue by beville** [comictagger/comictagger#607]
Using the latest build v1.6.0a12.dev0 (but I think before this), when I click on the "MD" column at the top, the app will catch a log a number of errors, and quit.  The number of logs seems about the same at items in the list.

```
2024-02-20T16:26:53 | comictagger | CRITICAL | Uncaught exception: TypeError: '<' not supported between instances of 'NoneType' and 'NoneType'
Traceback (most recent call last):
  File "/home/me/.local/lib/python3.10/site-packages/comictaggerlib/fileselectionlist.py", line 40, in __lt__
    return self.data(QtCore.Qt.ItemDataRole.UserRole) < other.data(QtCore.Qt.ItemDataRole.UserRole)  # type: ignore
TypeError: '<' not supported between instances of 'NoneType' and 'NoneType'
```

My own use case for sorting by tagged/untagged to easily see and group together what is not been tagged in the list.

Also, It's not a big deal for me, but the new combined metadata column makes it not feasible to sort by the existence of individual types.  (At least I think so; I'm not sure sure what would happen if it didn't crash).

**Comment by lordwelch** [comment]
I'm not sure how I would make that work without also potentially having an unlimited amount of columns. The exception will be fixed soon

**Comment by lordwelch** [comment]
Fixed in 5738433c2b7a15aac4ebf7fd1a279d47833aca5e

**Comment by lordwelch** [comment]
Fixed in [1.6.0-alpha.12]

[1.6.0-alpha.12]: https://github.com/comictagger/comictagger/releases/tag/1.6.0-alpha.12
